tests/integration/records/single_db/test_records_load_df.py

```diff
@@ -2,7 +2,6 @@
 import logging
 from .base_records_test import BaseRecordsIntegrationTest
 import datetime
-from odictliteral import odict
```

Comment:
This isn't needed with the modern Python which runs the integration tests, so one less thing needed as a dependency for them.